.env

```diff
@@ -9,7 +9,9 @@ SENTRY_IMAGE=getsentry/sentry:nightly
 SNUBA_IMAGE=getsentry/snuba:nightly
 RELAY_IMAGE=getsentry/relay:nightly
 SYMBOLICATOR_IMAGE=getsentry/symbolicator:nightly
+VROOM_IMAGE=getsentry/vroom:latest
```

Comment:
wait.. whaaaat? 🫨

Reply:
This should be `nightly`, for testing purposes i changed it but now we are finally publishing `nightly` tags properly.

Reply:
Ah ok. Understandable.